packages/syft/src/syft/grid/client/client.py

```diff
@@ -101,9 +101,13 @@ def login(
     password: Optional[str] = None,
     conn_type: Type[ClientConnection] = GridHTTPConnection,
     verbose: Optional[bool] = True,
-    timeout: Optional[int] = 120,
+    timeout: Optional[float] = None,
+    retry: Optional[int] = None,
 ) -> Client:
 
+    retry = retry if retry else 5  # Default to 5 retries
```

Comment:
MAYBE: set the default retry and timeout value in argument parameters

Reply:
instead of None

Reply:
cause we're using Optional typing which enforces that the value should be None. Therefore setting is later in the code.